app.js

```diff
@@ -60,25 +61,33 @@ addFriendForm.addEventListener('submit', (e) => {
     const formData = new FormData(addFriendForm);
 
     // > create a new friend, with a "name" property that
-    // is populated from `formData.get('name')` and a
-    // "satisfied" property with an initial value of 0
-
+    const friend = {
+        // is populated from `formData.get('name')` and a
+        name: formData.get('name'),
+        // "satisfied" property with an initial value of 0
+        satisfied: 0,
+    };
     // > add the new friend to the friends array
+    friends.push(friend);
 
     // > set the message state to let the user know
     // they invited a new friend to the festival, include the friend's
     // name in the message
+    message = `${friend.name} Has Been Invited To The Hunt`;
 
     addFriendForm.reset();
-
     // > call the display functions that need to re-display
+    displayFriends();
 });
 
 sayGoodbyeButton.addEventListener('click', () => {
     const stillHungry = [];
     for (const friend of friends) {
         // > if the friend is not fully satisfied, push
-        // them into the stillHungry array
+        if (friend.satisfied < 3) {
+            // them into the stillHungry array
+            stillHungry.push(friend);
+        }
```

Comment:
creating a new array to filter items from the old array that match a certain condition